assets/stylesheets/all.css.scss

```
@@ -22,3 +22,10 @@
 
 // Modules
 @import '_modules/**/*';
+
+// Custom
+a.gitter-open-chat-button,  a.gitter-open-chat-button:hover {
```

```suggestion
a {
  &.gitter-open-chat-button {
    &, &:hover {
```

that would be the "sassy" way - but that's more like a personal taste I guess

---

Seems unnecessary to nest for something so small IMO since this is just an override and not part of the greater system.

Only comment would be to remove the `a` selectors since that increase specificity.

---

Yes you're absolutely right. I'd even say it's way more readable when sticking to the plain CSS.